tests/tools/oecertdump/host/host.cpp

```diff
@@ -228,24 +210,25 @@ static int _parse_args(int argc, const char* argv[])
 
     while (i < argc)
     {
-        if (strcmp(DEFAULT_OUT_FILE, argv[i]) == 0)
+        if (strcmp(INPUT_PARAM_OUT_FILE, argv[i]) == 0)
         {
-            if (argc >= i + 1)
+            if (argc >= i + 2)
```

Comment:
Since argv[i] itself (0-index) is the (i+1)th argument (1-index), we should check if there is a (i+2)th argument.